Makefile

```diff
@@ -5,20 +5,20 @@ docker-nearcore-nightly:
 	DOCKER_BUILDKIT=1 docker build -t nearcore-nightly -f Dockerfile.nightly .
 
 release:
-	cargo build -p neard --release
-	cargo build -p keypair-generator --release
-	cargo build -p genesis-csv-to-json --release
-	cargo build -p near-vm-runner-standalone --release
-	cargo build -p state-viewer --release
-	cargo build -p store-validator --release
+	cargo +stable build -p neard --release
+	cargo +stable build -p keypair-generator --release
+	cargo +stable build -p genesis-csv-to-json --release
+	cargo +stable build -p near-vm-runner-standalone --release
+	cargo +stable build -p state-viewer --release
+	cargo +stable build -p store-validator --release
 
 debug:
-	cargo build -p neard
```

Comment:
Cannot we have set of options extracted to the variable?

Reply:
Can you explain more?  I'm not familiar with Makefiles.

Reply:
Just create variable `RUST_OPTIONS`, set it to `+stable` and use `$RUST_OPTIONS` later on.